cpp/modmesh/buffer/pymod/array_common.hpp

```diff
@@ -95,25 +95,30 @@ class ArrayPropertyHelper
 
         if (args.size() == 2)
         {
+            const bool isNumber = py::isinstance<py::bool_>(args[1]) || py::isinstance<py::int_>(args[1]) || py::isinstance<py::float_>(args[1]);
```

Comment:
better way to identify a number

Reply:
It might run faster by caching `args[1]` locally in a `py::object`.  Every time you do the indexing, some Python operations incur.  But Python is already slow so this is not really a big deal.

Reply:
But I do want local variables (e.g., `isNumber`) to use `snake_case`, which is consistent to the existing code in this function.  Please change the style.

Reply:
I thought `py::args` underlying is `py::object`, but actually it's `py::argument_record`.

Fixed in f9f90021927345266d1465e7c6169eeac2b58aad